_posts/2021-04-21-security-incident-disclosure.md

```diff
@@ -0,0 +1,29 @@
+---
+title: Security Incident Disclosure
+author: reitermarkus
+---
+
+On 18th April 2021, [a security researcher identified](https://blog.ryotak.me/post/homebrew-security-incident-en/) a vulnerability in our `review-cask-pr` GitHub Action used on the `homebrew-cask` and all `homebrew-cask-*` taps (non-default repositories) in the Homebrew organization.
+
+Whenever an affected cask tap received a pull request to change the only version of a cask, the `review-cask-pr` GitHub Action would automatically review and approve the pull request. The approval would then trigger the `automerge` GitHub Action which would merge the approved pull request. This was reported via HackerOne and [a proof-of-concept (PoC) pull request demonstrating the vulnerability](https://github.com/Homebrew/homebrew-cask/pull/104191) was submitted with our permission. We subsequently [reverted the PoC pull request](https://github.com/Homebrew/homebrew-cask/pull/104197), [disabled and removed the `automerge` GitHub Action](https://github.com/Homebrew/homebrew-cask/pull/104199) and [disabled and removed the `review-cask-pr` GitHub Action](https://github.com/Homebrew/homebrew-cask/pull/104204) from all vulnerable repositories.
```

Comment:
```
Whenever an affected cask tap received a pull request to change the only version of a cask, the `review-cask-pr` GitHub Action would automatically review and approve the pull request. The approval would then trigger the `automerge` GitHub Action which would merge the approved pull request. This was reported via HackerOne 
```

Isn't this `this` should mention the `a vulnerability in our `review-cask-pr` GitHub Action`?